utils/clone-llvm.sh

```diff
@@ -14,14 +14,40 @@
 #
 ##===----------------------------------------------------------------------===##
 
+# The LLVM commit the project depends on
 export commithash=11c3b979e6512b00a5bd9c3e0d4ed986cf500630
+# It is not clear why we need a branch
 branch=air-2022.12
 
-git clone --depth 1 https://github.com/llvm/llvm-project.git llvm
-pushd llvm
-git fetch --depth=1 origin $commithash
-git checkout $commithash -b $branch
+here=$PWD
+
+# Use --llvm-worktree <directory-of-local-LLVM-repo> to reuse some existing
+# local LLVM git repository
+if [ x"$1" == x--llvm-worktree ]; then
```

Comment:
Personally when I have an existing llvm repo I simply don't run this script (actually I never run this script...). I wonder if this just adds complexity for a personal git preference?